spec/fixtures/cookbooks/oneview_test_api500_synergy/metadata.rb

```diff
@@ -0,0 +1,7 @@
+name             'oneview_test_api500_synergy'
+maintainer       'none'
+license          'All rights reserved'
```

Comment:
Shouldn't this be `license 'Apache v2.0'?

Reply:
It is a fake testing cookbook, it's a fixture